src/Engine/GameEvents/MoveEvent.java

```diff
@@ -49,8 +49,7 @@ public void performEvent(MapMaster gameMap)
         gameMap.moveUnit(unit, endpoint.x, endpoint.y);
       unit.isTurnOver = true;
 
-      if( unit.model.needsFuel )
-        unit.fuel = Math.max(0, unit.fuel - fuelBurn); // Don't prevent zero-distance "moves" when out of fuel.
+      unit.fuel = Math.max(0, unit.fuel - fuelBurn);
```

Comment:
I assume you verified this doesn't break zero-distance moves?

Reply:
Seems fine to me, boss.